prefect_gcp/workers/vertex.py

```diff
@@ -433,7 +433,10 @@ async def run(
             )
 
         error_msg = final_job_run.error.message
-        if error_msg:
+
+        # Vertex will include an error message upon valid
+        # flow cancellations, so we'll avoid raising an error in that case
+        if error_msg and "CANCELED" not in error_msg:
```

Comment:
had to exclude cancellation messages, as this will throw the runtime error

![image](https://github.com/PrefectHQ/prefect-gcp/assets/19556538/9960fb8e-7efa-4fc1-9446-62b4d6063b82)